src/BitByteNode.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './BitByteNode.css'; // Dummy CSS that will be changed later
+
+/** 
+ * Representes a node in the bit-byte tree
+ * Declared as a class component for potential additional functions 
```

Comment:
What additional functions did you need from a class component over a functional one?

Reply:
As of right now, I don't think there's a need of functions anymore, because we no longer need the addBit() function.
We might need functions like ComponentDidMount() in the future? But not right now

Reply:
For ComponentDidMount() it's somewhat the same as the useEffect() hook with a functional component. https://reacttraining.com/blog/useEffect-is-not-the-new-componentDidMount/

Reply:
Should we try to be consistent and only use functional components exclusively? I'm not sure what class components have over functional components now that hooks are a thing

Reply:
Sounds good, just pushed change to make everything a functional component